### PR TITLE
added flag to trigger upgrades during deployment

### PR DESCRIPTION
**Pull request by andywis**

### CODE REVIEW

**Comment by andywis** [comment]
Tested; working.

File: openshift-template.yaml [hunk 1]
```diff
@@ -112,6 +112,10 @@ parameters:
     type: boolean
     description: download certificates with certbot during install
     default: false
+  do_upgrades:
+    type: boolean
+    description: Whether to perform upgrade as part of Ansible deployment
```
Comment (stevemul) [inline openshift-template.yaml]: Can we be slightly more specific here? What upgrades are we doing? In this case its Operating system package upgrades for example

File: openshift-template.yaml [hunk 2]
```diff
@@ -112,6 +112,10 @@ parameters:
...
+    default: false
```
Comment (stevemul) [inline openshift-template.yaml]: I think this should default to true personally - better to accidentally patch to latest than to deploy on older packages? Then we have to choose not to if we want faster tests.

**Comment by andywis** [comment]
addressed comments from @stevemul

**Pull request merged**